js-api-doc/value/calculation.d.ts

```diff
@@ -0,0 +1,141 @@
+import {List, ValueObject} from 'immutable';
+import {Value, SassNumber, SassString} from './index';
+
+/** The type of values that can be arguments to a SassCalculation. */
+type CalculationValue =
```

Comment:
This should be exported, so other libraries can refer to it as necessary. It should also (along with any other top-level types here) be in `@category Custom Function`.